install/kubernetes/export-doc.sh

```diff
@@ -47,7 +47,7 @@ The Tetragon Helm chart source is available under
 and is distributed from the Cilium helm charts repository [helm.cilium.io](https://helm.cilium.io).
 
 To deploy Tetragon using this Helm chart you can run the following commands:
-```shell-session
+```shell
```

Comment:
Thanks! well spotted, the new theme uses `shell` everywhere, really well spotted it just changed.

Reply:
heh, `install/kubernetes/tetragon/test.sh` was actually failing without this change :)